lib/cluster.js

```diff
@@ -167,9 +193,19 @@ function addListeners() {
 					if (request.failed) return; // Callback already run with Error.
 
 					const registry = AggregatorRegistry.aggregate(request.responses);
+					const g = new Gauge({
+						name: 'nodejs_prom_client_cluster_workers',
+						help: 'Number of connected cluster workers reporting to prometheus',
+						registers: [registry]
+					});
+					g.set(request.workerCount);
```

Comment:
Might want to move this new metric to a separate PR. (If @siimon and @SimenB are okay with it, it's a simple change that could land before this PR is ironed out.)

Reply:
Noted, however this is very specific to the coordinated option - perhaps it should be exposed only when coordinated is true. Perhaps a similar metric could be added to the default metrics collected that counts *all* the cluster workers?